src/styles/partials/components/_utilities.scss

```diff
@@ -0,0 +1,25 @@
+.visuallyhidden,
+h2 {
```

Comment:
h2 should be removed, we only want the visually hidden class.

Reply:
removed